src/test/fuzz/txgraph.cpp

```diff
@@ -721,7 +729,18 @@ FUZZ_TARGET(txgraph)
                 break;
             } else if (command-- == 0) {
                 // DoWork.
-                real->DoWork();
+                uint64_t iters = provider.ConsumeIntegralInRange<uint64_t>(0, alt ? 10000 : 255);
+                if (real->DoWork(iters)) {
```

Comment:
would it be possible to assert that this must return true if the value is high enough since we have a limited txgraph size?